src/components/page/blocks/WidgetBlock.jsx

```diff
@@ -26,6 +27,7 @@ class WidgetBlock extends React.PureComponent {
       HubblePlot,
       HubblePlot3D,
       GalacticProperties,
+      HeatMap3DPlot,
```

Comment:
Please rename this Class and filename to something like LargeScaleStructureSOMETHING instead of HeatMapSOMETHING as these are not heat maps, and we may have other components in the future that are.